libs/s25main/desktops/dskSplash.cpp

```diff
@@ -76,6 +77,9 @@ void dskSplash::LoadFiles()
 
     } else
     {
+        VIDEODRIVER.showMessageBox("Please install Settlers II game files into",
```

Comment:
I'd rename this to `showErrorMessage` as that is what you implemented. 
Then better put the call below `s25util::error`

Reply:
and please do logging before the message box. if a user aborts the program due to the message box, the log entry will never be written.
We also should be more explicit whats missing.

Reply:
See
>  Then better put the call below `s25util::error`